### DISCUSSION TITLE
Easy ways to enable extra notifications and/or check process is running?

### DISCUSSION THREAD

**Discussion by johnsidoli** [abraunegg/onedrive#1778] [Ideas]
I see the client already notifies on:

-     Aborting a sync if .nosync file is found
-     Cannot create remote directory
-     Cannot upload file changes
-     Cannot delete remote file / folder
-     Cannot move remote file / folder

Is there a way I can enable system notifications to come up for everything the client does... or at least more stuff like

- uploading a file e.g. "uploaded two files"
- downloading a file(s) e.g. "downloaded 3 files)
- deleting a file(s) e.g. "deleted 5 files"

and stuff like that?

My reason for this is just that I want to see it is working when I hit save on stuff and/or delete stuff and/or do anything really... just as I test and get used to it (it's scary not seeing stuff happen and not knowing if it successfully synced stuff!).

**Comment by abraunegg** [comment]
@johnsidoli 
> Is there a way I can enable system notifications to come up for everything the client does...

Not without code changes, but then it would also 'flood' the desktop notifications daemon and cause other untested side effects.

> My reason for this is just that I want to see it is working when I hit save on stuff and/or delete stuff and/or do anything really... just as I test and get used to it (it's scary not seeing stuff happen and not knowing if it successfully synced stuff!).

Enable logging for the application to a separate file - then, use `tail -f <log_file>` to watch what the client is doing in relation to uploads / downloads etc - this is simpler and available to you right now.
